packages/components/src/public/QueryColumn.ts

```diff
@@ -324,7 +328,7 @@ export class QueryColumn implements IQueryColumn {
     }
 
     isUnitsLookup(): boolean {
-        return this.isLookup && this.lookup.queryName === QueryColumn.MEASUREMENT_UNITS_QUERY;
+        return this.isLookup() && this.lookup.queryName === QueryColumn.MEASUREMENT_UNITS_QUERY;
```

Comment:
Surprising that this didn't cause more errors to be logged somewhere...

Reply:
I was surprised too...